compiler/luci/pass/include/luci/CircleOptimizer.h

```diff
@@ -32,9 +32,10 @@ class CircleOptimizer final
   {
     enum Algorithm
     {
+      FuseBCQ,
       FuseInstanceNorm,
       ResolveCustomOpBatchMatMul,
-      QuantizeWithMinMax
+      QuantizeWithMinMax,
```

Comment:
This is not related with this PR but I've requested this change as to make later changes not edit last line...